standalone/build.gradle.kts

```diff
@@ -0,0 +1,115 @@
+import com.replaymod.gradle.preprocess.PreprocessTask
+
+plugins {
+    kotlin("jvm")
+    id("gg.essential.defaults")
+    id("gg.essential.defaults.maven-publish")
+}
+
+val parent = evaluationDependsOn(project.parent!!.path)
+
+group = "gg.essential"
+version = parent.version
+base.archivesName = "universalcraft-standalone"
+kotlin.compilerOptions.moduleName = "universalcraft-standalone"
+publishing.publications.named<MavenPublication>("maven") { artifactId = "universalcraft-standalone" }
+java.withSourcesJar()
+kotlin.jvmToolchain(8)
+
+dependencies {
+    api(kotlin("stdlib-jdk8", "2.0.20-RC"))
+    api("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.9.0-RC")
+
+    implementation("dev.folomeev.kotgl:kotgl-matrix:0.0.1-beta")
+
+    val lwjglVersion = "3.3.3"
+    val lwjglModules = listOf("lwjgl", "lwjgl-glfw", "lwjgl-opengl", "lwjgl-stb", "lwjgl-nanovg")
+    val lwjglNatives = listOf("linux", "macos", "macos-arm64", "windows")
+    api(platform("org.lwjgl:lwjgl-bom:$lwjglVersion"))
+    for (module in lwjglModules) {
+        api("org.lwjgl:$module:$lwjglVersion")
```

Comment:
You addded the bom, why specify `$lwjglVersion`?

Reply:
Copy/pasted from another place where we don't use the bom, and then forgot to clean it up after adding the bom.